### ISSUE TITLE
Copied Code is found in some of the data-structure

### ISSUE THREAD

**Issue by rishav-pandey** [OpenGenus/cosmos#2515]
**This is a(n):**

- [ ] New algorithm
- [ ] Update to an existing algorithm
- [x] Error
- [ ] Proposal to the Repository

**Details:**

I was reviewing some of the files and found that many of the codes are just been copied from internet and pasted here.
So please just don't copy the code and paste to show your contribution here. Your self made code and algorithm is only accepted here.

Thank You...!!!!

**Comment by AdiChat** [comment]
Yes, this is a serious problem and few days back, we identified few such problems.

Thereon, we have taken this seriously in our review process. Though we did put a few contributions on hold due to such sources, certainly a few did slip under our eye. 

We hope to make management of Cosmos better and easier than ever before.

**Comment by the-ethan-hunt** [comment]
IMHO, many contributors here are those who have recently joined open-source and maybe coding itself. Though they have copied the code of some algorithms, they have a "dictionary" that is not having other copied codes and this will help them to understand and develop their own code. So, maybe we should give a blind eye to such contributors. 
I hope that OpenGenus allows itself to be more beginner-friendly

**Comment by AnEmortalKid** [comment]
Are we really helping by allowing copy pasted code that sometimes doesn't even compile?
